client/src/components/game/cards/CardReverse.jsx

```diff
@@ -0,0 +1,16 @@
+import './cardReverse.scss';
+
+import React from 'react';
+
+const CardReverse = ({ color, rightOffset, bottomOffset, text }) => {
+  return (
+    <div
+      className={'card-reverse ' + color}
+      style={{ right: rightOffset, bottom: bottomOffset }}
```

Comment:
easier to achieve with styled-components

Reply:
Do I understand correctly, that we would need to declare an `Offset` styled component with `right: rightOffset, bottom: bottomOffset` and then pass `Offset` as a third class to `className`?

Reply:
I was thinking about something like
```javascript
const StyledCardReverse = styled.div`
  // some styles
  right: ${props => props.rightOffset}
  bottom: ${props => props.bottomOffset}
`
const CardReverse = ({ color, rightOffset, bottomOffset, text }) => {
  return (
    <StyledCardReverse>
      <span>{text}</span>
    </StyledCardReverse>
  )
```

Reply:
but sadly to use color from theme we would need to create something like theme provider for styled components